### DISCUSSION TITLE
Make issue forms available for pull requests

### DISCUSSION THREAD

**Discussion by SimonBrandner** [community/community#4881] [Projects and Issues]
In my opinion, It would be great if you could use issue forms for pull requests too. I think this is especially useful for open source projects where not all contributors have the right to do triage. I see the following uses:

+ Sometimes it would be nice to assign reviewers (code, design, product...) based on what's filled in the form
+ For projects that generate changelogs using pull requests, it would be great to choose a changelog section (bug, feature, internal...) based on the form content
+ And probably others...

**Comment by lukehefson** [comment]
Thanks for the feedback @SimonBrandner – I’ll be sure to pass it on to the relevant folk!

**Reply by Bibo-Joshi** [reply]
Adding them to discussions would also be neat :) See also #3709 and #2838

**Comment by jeremystretch** [comment]
I'll chime in with another use case. Because of our project's size, we require that every proposed change first be discussed in an issue. If approved, a pull request can then be submitted. This is done to help prevent people from wasting time working on changes that would ultimately be rejected.

While we do employ a pull request template to indicate the need to reference an existing issue when submitting a PR, some people still ignore it. Turning this into a required form field would help cut down on the noise a bit.

**Comment by dillingham** [comment]
And values available in actions

**Comment by MominRaza** [comment]
@lukehefson No update on this,
It will be very helpful for new joiners in our team, if we can make form fields mandatory.

**Reply by lukehefson** [reply]
@MominRaza I left GitHub a while back I'm afraid. I personally would love to see forms for pull requests though in my new job!
